scripts/vscripts/tf2ware_ultimate/minigames/pick_up_that_can.nut

```diff
@@ -0,0 +1,222 @@
+minigame <- Ware_MinigameData
+({
+	name          = "Pick Up That Can"
+	author		  = "CzechMate"
+	description   = "Pick Up And Trash The Can!"
+	duration      = 7.0
+	music         = "sweetdays"
+})
+
+::trashcan_model <- "models/props_trainstation/trashcan_indoor001b.mdl"

```

Comment:
Do not put variables into global scope in minigames, like with the :: here. Declare them as normal variables (i.e. without the ::)